styleguide/_themes/global/scss/forms.scss

```diff
@@ -114,3 +114,51 @@ table {
 .table > thead > tr > th {
   font-weight: unset;
 }
+
+.formModal-closeIcon {
+  opacity: .7;
+}
+
+.formModal-title {
+  text-align: center;
+}
+
+.formModal-content {
+  border-radius: 2px;
+  padding: 15px;
+}
+
+.formModal-header {
+  border-bottom: 0;
+}
+
+.formModal-body {
+  padding: 10px 20px;
+}
+
+.formModal-dialog {
+
+  @media (min-width: $screen-sm-min) {
+    width: 440px;
```

Comment:
@amberRrucker What do you recommend here? All bootstrap modals have fixed widths up until it hits tablet/mobile, at which point it goes fluid. If we go percentage width on desktop, it's either going to look too wide or too small depending on the screen size.